src/profile/MfaCardLabel.vue

```diff
@@ -35,16 +35,19 @@
 </template>
 
 <script>
-import { change } from '@/global/mfa'
+import { changeWebauthn } from '@/global/mfa'
 
 export default {
   props: {
     label: {
       type: String,
     }, 
-    id: {
+    keyId: {
       type: Number,
-    }, 
+    },
+    mfaId: {
```

Comment:
Does this get hydrated properly so that it doesn't end up as `undefined` down in the call to `changeWebauthn()`?